src/app/modules/verification/verification.model.ts

```diff
@@ -115,10 +115,8 @@ const compileVerificationModel = (db: Mongoose): IVerificationModel => {
     return fields
   }
 
-  const getVerifiablePaymentContactField = (
-    paymentsField?: FormPaymentsField,
-  ) => {
-    if (!paymentsField?.enabled) return null
```

Comment:
to ensure a smooth change of fields, we can do a 2-step roll out, by first introducing the new field, and keep the existing field. And at the 2nd step, we can remove the existing(deprecated) field entirely. In summary:

- current, existing prod version: FE sends supports paymentEnabled field, and BE receives and checks paymentEnabled field
- upcoming release (supports both fields, but make paymentEnable optional): FE sends both supports paymentEnabled field and paymentAccConnected field, and BE checks for optional paymentEnabled field and paymentAccConnected field (ie `paymentsField.enabled OR paymentsAccConnected)
- subsequent release (remove enable field entirely): FE sends only paymentAccConnected field, and BE still checks for only paymentsAccConnected field 

Doing so, we are not pressured to run the script, and *if* there is a need of rollback, we are still backward compatible. And it's a good practice for a smooth rollout.